cfn-resources/cluster/docs/tag.md

```diff
@@ -1,6 +1,6 @@
 # MongoDB::Atlas::Cluster tag
 
-Advanced configuration details to add for one cluster in the specified project.
```

Comment:
why has this file changed?

Reply:
Checking the changes to the json schema. It looks like when the `tag` attribute was introduced: https://github.com/mongodb/mongodbatlas-cloudformation-resources/pull/671/files#diff-4d8655df51997e3e7feef9b011dd0c6e63090c3b6c4cfa1b0156bc072bda594a
They didn't use the generation or perhaps cfn documentation generation has changed 🤔

Reply:
not for this PR, but I'm thinking we could do some step in GH to fail if there is any file change after applying make build (like we did for mocking in TF)